TempoCore/Source/TempoCore/Private/TempoWorldSettings.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright Tempo Simulation, LLC. All Rights Reserved
+
+#include "TempoWorldSettings.h"
+
+#include "TempoCore.h"
+#include "Engine/DirectionalLight.h"
+#include "Kismet/GameplayStatics.h"
+
+namespace
+{
+	// These values result in relatively consistent perceived brightness in the range of ~5-100 Lux of scene intensity.
+	constexpr float ExposureBiasToBrightnessScale = 30.0 / 150.0;
+	constexpr float ExposureBiasToBrightnessOffset = -15.0;
+}
+
+void ATempoWorldSettings::SetDefaultAutoExposureBias()
+{
+	check(GetWorld());
+	if (const ADirectionalLight* DirectionalLight = Cast<ADirectionalLight>(UGameplayStatics::GetActorOfClass(GetWorld(), ADirectionalLight::StaticClass())))
```

Comment:
This works for `ADirectionalLight` Actors.  But, `TempoSunSky` has a DirectionalLight as a SceneComponent.  So, we'll need to check for DirectionalLights as SceneComponents to catch it.